packages/terra-application-layout/src/menu/_ApplicationMenu.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import AppDelegate from 'terra-app-delegate';
+import ApplicationMenuLayout from 'terra-application-menu-layout';
+import { ApplicationMenuName } from 'terra-application-name';
+import RoutingStackDelegate from 'terra-navigation-layout/lib/RoutingStackDelegate';
+import { processedRoutesPropType } from 'terra-navigation-layout/lib/configurationPropTypes';
+import { ApplicationMenuUtility } from 'terra-application-utility';
+import { disclosureType as modalDisclosureType } from 'terra-modal-manager';
+import { availableDisclosureSizes } from 'terra-disclosure-manager';
+
+import 'terra-base/lib/baseStyles';
+import ApplicationLayoutPropTypes from '../utils/propTypes';
+import Helpers from '../utils/helpers';
+
+import UtilityMenuWrapper from './_UtilityMenuWrapper';
+
+import styles from './ApplicationMenu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The AppDelegate instance that will be propagated to the components presented within the NavigationLayout.
+   */
+  app: AppDelegate.propType,
+  /**
+   * The element to be placed within the fill flex styled content area.
+   */
+  content: PropTypes.element,
+  /**
+   * The element to be placed within the fit top area for extensions within the layout.
+   */
+  extensions: PropTypes.element,
+  /**
+   * The Object of layout-related APIs provided to the components of the Layout.
+   */
+  layoutConfig: ApplicationLayoutPropTypes.layoutConfigPropType.isRequired,
+  /**
+   * Configuration values for the ApplicationName component.
+   */
+  nameConfig: ApplicationLayoutPropTypes.nameConfigPropType,
+  /**
+   * Delegate prop that is provided by the NavigationLayout.
+   */
+  routingStackDelegate: RoutingStackDelegate.propType.isRequired,
+  /**
+   * The set of routes currently identified by the NavigationLayout. This prop is provided by the NavigationLayout.
+   */
+  navigationLayoutRoutes: PropTypes.arrayOf(processedRoutesPropType),
```

Comment:
I'm not sure this renders correctly in the props-table.

Reply:
It doesn't. Brett recommended removing it and beefing up the custom docs, and I'm going to do that.